src/workspaces/dashboard/RightBoxSection.ts

```diff
@@ -8,18 +8,24 @@ import * as Style from 'src/libs/style';
 import { useLocalPref } from 'src/libs/useLocalPref';
 import { WorkspaceWrapper } from 'src/workspaces/utils';
 
-interface RightBoxSectionProps {
-  title: string;
+interface WorkspaceRightSectionProps {
   workspace: WorkspaceWrapper; // used for metrics eventing
```

Comment:
I'm not following-- what is the point of putting `workspace` inside of `WorkspaceRightSectionProps`?

Reply:
If you are trying to say that other usages of `RightBoxSection` don't have a workspace, then `WorkspaceRightSectionProps` should extend `RightBoxSectionProps` and add a workspace to it.

Reply:
However, `RightBoxSection` is in the workspaces package. It makes sense that it requires a workspace.

If you want to reuse `RightBoxSection`, you should move it out of this package into a common component package.

Reply:
This was done in an attempt the make the component re-usable. `workspace` was previously a required field in the props, however it is not required for other components, such as on the Workflow Details page. It didn't feel right to make `workspace` optional, as I wanted to avoid messing with the metrics if a component is in fact being used in association with a workspace.

Reply:
Right, if you are going to reuse this outside of workspaces you need to make it an actual reusable component. Details like having a workspace and emitting metrics wouldn't belong in the reusable component.